src/components/UploadSource/UploadSource.md

```diff
@@ -0,0 +1,147 @@
+This component helps to upload bible books
+
+### Upload Source
+
+```js
+import UploadSource from './UploadSource';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import { API } from '../../store/api';
+import SnackBar from '../SnackBar/SnackBar.js';
+
+const [selectedFiles, setSelectedFiles] = React.useState('');
+const [responseStatus, setResponseStatus] = React.useState([]);
+
+const handleClose = () => {
+  setResponseStatus([false]);
+};
+
+const [projectData, setProjectData] = React.useState({
+  projectId: 100000,
+  projectName: 'Mission Agape',
+  sourceLanguage: {
+    languageId: 100037,
+    language: 'English',
+    code: 'en',
+    scriptDirection: 'left-to-right',
+    metaData: {
+      region: 'United Kingdom, Europe',
+      'alternate-names': [
+        'Anglit',
+        'Kiingereza',
+        'Gustavia English',
+        'Samaná English',
+        'Saint Lucian English',
+        'Noongar',
+        'Noonga',
+        'Newcastle Northumber',
+        'Neo-Nyungar (Noogar)',
+        'Glaswegian',
+        'Brummy',
+        'Birmingham (Brummie)',
+        'Bay Islands English',
+        'Australian Standard English',
+        'Aboriginal English',
+        'African American Vernacular English (AAVE)',
+      ],
+      'suppress-script': 'Latn',
+      'is-gateway-language': true,
+    },
+  },
+  targetLanguage: {
+    languageId: 100057,
+    language: 'Hindi',
+    code: 'hi',
+    scriptDirection: 'left-to-right',
+    metaData: {
+      region: 'India, Asia',
+      'alternate-names': [
+        'Khadi Boli',
+        'Khari Boli',
+        'Dakhini',
+        'Hindi-Urdu',
+        'Khariboli',
+      ],
+      'suppress-script': 'Deva',
+      'is-gateway-language': true,
+    },
+  },
+  documentFormat: 'usfm',
+  users: [
+    {
+      project_id: 100000,
+      userId: 10101,
+      userRole: 'owner',
+      metaData: null,
+      active: true,
+    },
+  ],
+  metaData: {
+    books: ['mat', 'luk', 'jhn', '3jn'],
+    useDataForLearning: true,
+  },
+  active: true,
+});
+
+const clearState = () => {
+  setSelectedFiles('');
+};
+
+const readFileAsText = (file) => {
```

Comment:
Ideally this upload logic should also be in the component as its boiler plate code , the projectId can be passed as a prop in that case, but its fine if its here also they can copy and use it from here as well.